support-modules/retry/build.sbt

```
@@ -0,0 +1,5 @@
+import LibraryVersions.circeVersion
+
```

For this to be picked up in the main build, don't you need to add the module name [like other modules](https://github.com/guardian/support-frontend/blob/main/support-modules/aws/build.sbt#L3-L5)?

```suggestion

name := "module-retry"

description := "Scala library to retry asynchronous methods if they fail."

```

---

Thanks for the detail!